### PR TITLE
document that enabling manageJobsWithoutQueueName can be risky

### PR DESCRIPTION
**Pull request by dgrove-oss**
#### What type of PR is this?
/kind documentation

#### What this PR does / why we need it:

Document issue #2277.  Didn't mark as fixing so that the issue won't be auto-closed just because we document the problem.

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Review by mimowo — COMMENTED** [review]
/lgtm
/assign @alculquicondor

File: site/content/en/docs/installation/_index.md [hunk 1]
```diff
@@ -151,9 +151,13 @@ data:
 __The `integrations.externalFrameworks` field is available in Kueue v0.7.0 and later.__
 
 > **Note**
-> See [Sequential Admission with Ready Pods](/docs/tasks/setup_sequential_admission) to learn
+> See [Sequential Admission with Ready Pods](/docs/tasks/manage/setup_sequential_admission) to learn
```
[the remark this answers is not in the document]
Comment (alculquicondor) [reply site/content/en/docs/installation/_index.md:154]: while at it, could you apply this suggestion? https://github.com/kubernetes-sigs/kueue/issues/1755

File: site/content/en/docs/installation/_index.md [hunk 2]
```diff
@@ -151,9 +151,13 @@ data:
...
 more about using `waitForPodsReady` for Kueue.
 
+> **Note**
+> Setting `manageJobsWithoutQueueName` to `true` without disabling the `batch/job` integration
```
Comment (alculquicondor) [inline site/content/en/docs/installation/_index.md:158]: This sounds too specific to the way OpenShift conducts operations.

Maybe you can prephrase like this:
Certain Kubernetes distributions might use batch/jobs to perform maintenance operations. For these distributions, setting ......

File: site/content/en/docs/installation/_index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (dgrove-oss) [reply site/content/en/docs/installation/_index.md]: Done.

File: site/content/en/docs/installation/_index.md (hunk 2 above)
Comment (dgrove-oss) [reply site/content/en/docs/installation/_index.md]: Done.

**Review by tenzen-y — COMMENTED** [review]
Thank you.
/lgtm
/aaprove

**Pull request merged**